src/sentry/issues/endpoints/group_details.py

```diff
@@ -271,7 +271,7 @@ def get(self, request: Request, group: Group) -> Response:
                 )
                 integration_issues = serialize(
                     external_issues,
-                    request,
+                    request.user,
```

Comment:
lol, this was just wrong -- fortunately nothing needed `user` in serializing this (and the one below!)